backend/src/model/oianalytics-command.model.ts

```diff
@@ -88,6 +88,16 @@ export interface OIBusDeleteNorthConnectorCommand extends BaseOIBusCommand {
   northConnectorId: string;
 }
 
+export interface OIBusCreateOrUpdateSouthConnectorItemsFromCSVCommand extends BaseOIBusCommand {
+  type: 'create-or-update-south-items-from-csv';
+  southConnectorId: string;
+  commandContent: {
+    deleteItemsNotPresent: boolean;
+    csvContent: string;
+    delimiter: string;
+  };
```

Comment:
it is not wrapped in a commandContent is it?

Reply:
Not from OIAnalytics, but I wrap it in OIBus when received, in a similar fashion than OIBus update commands. Should I created dedicated fields instead (for both commands)?

Reply:
not sure why you would transform the original signature, what does this bring?

Reply:
nevermind it is oibus model forget what i said